src/it/java/org/checkstyle/suppressionxpathfilter/XpathRegressionUnusedLocalVariableTest.java

```diff
@@ -43,6 +43,7 @@ public void testOne() throws Exception {
 
         final DefaultConfiguration moduleConfig =
                 createModuleConfig(UnusedLocalVariableCheck.class);
+        moduleConfig.addProperty("allowUnnamedVariables", "false");
```

Comment:
I also disagree here. We should be checking both on and off can be suppressed by xpath.

The point of these tests is to ensure xpath always work. In the beginning, some checks and ASTs could not be suppressed.

Reply:
done